lib/migrate/loaders/loaders.js

```diff
@@ -219,7 +248,7 @@ module.exports = function(j, ast) {
 			.forEach(createArrayExpressionFromArray);
 
 	/**
-	 * Finds loaders with options encoded as query string and replaces it with options obejct
+	 * Find loaders with options encoded as a query string and replace the string with an options object
```

Comment:
interactive verb? `replaces`

Reply:
I'm a little confused about this. You want to change `replace` to `replaces` but leave `Find` in the same sentence? It seems like they should match. I skimmed through some of the other comments and it looks like sometimes the doclets use the `replace` version and sometimes they use the `replaces` version. If you want, I can make them all consistent. Or there may be something special about this one case that I don't understand—just let me know.

Reply:
its a small pick, so no worries 👍